docs/contributing-building.md

```diff
@@ -142,3 +142,30 @@ cargo build --target aarch64-unknown-linux-gnu --release
 The built executable will be located at
 `target/aarch64-unknown-linux-gnu/release/wasmtime`. Note that you can
 cross-compile the C API in the same manner as the CLI too.
+
+## Running a Cross-Compiled Wasmtime in qemu (emulation)
+
+Once you have cross-compiled a binary, it is possible to run it on an emulator
+if you do not have access to (or do not wish to use) hardware with the given
+architecture. This can be done using an emulator such as `qemu`. The `qemu`
+user-space emulation support allows running, for example, a Linux/aarch64
+binary on a Linux/x86-64 host, as long as you have the system libraries for
+aarch64 as well.
+
+To try this out, first install `qemu`, making sure that the user-space emulator
+option for your target architecture is enabled. On Debian-based Linux
+distributions (including Ubuntu), this is in the `qemu-user` package, for
+example.
+
+Next, make sure that you have system libraries for the target. You will already
+have these present if you cross-compiled as described above.
+
+Finally, you can run the `wasmtime` binary under `qemu`; the following example
+is for an `aarch64` target. Adjust the library paths as appropriate; these are
+correct for Ubuntu/Debian's cross-compilation packages.
+
+```shell
+QEMU_LD_PREFIX=/usr/aarch64-linux-gnu/ \
+  LD_LIBRARY_PATH=/usr/aarch64-linux-gnu/lib/ \
```

Comment:
I don't think I ever had to update this env var when I was testing, although is this required for you locally?

Reply:
It is at least on my setup (Arch Linux and its associated cross-build packages):

```plain
[cfallin@cfallin-laptop wasmtime]$ qemu-aarch64 target/aarch64-unknown-linux-gnu/release/wasmtime
/lib/ld-linux-aarch64.so.1: No such file or directory
[cfallin@cfallin-laptop wasmtime]$ qemu-aarch64 -L /usr/aarch64-linux-gnu/ target/aarch64-unknown-linux-gnu/release/wasmtime
target/aarch64-unknown-linux-gnu/release/wasmtime: error while loading shared libraries: libgcc_s.so.1: cannot open shared object file: No such file or directory
[cfallin@cfallin-laptop wasmtime]$ LD_LIBRARY_PATH=/usr/aarch64-linux-gnu/lib64/ qemu-aarch64 -L /usr/aarch64-linux-gnu/ target/aarch64-unknown-linux-gnu/release/wasmtime
wasmtime 0.16.0
Wasmtime WebAssembly Runtime
```

(on Arch the `.../lib64` vs `.../lib` differs but otherwise it appears Ubuntu should work the same)

It's possible this has to do with some rpath-related or other magic in the `.so`s; if there's a simpler incantation that works, all the better!

Reply:
Bummer! Dunno why it works on Ubuntu for me without this extra flag. Alas!